### PR TITLE
Review display

### PR DESCRIPTION
**Pull request by gaikwadsid**
## Description

[Link to Ticket](insert the link to your ticket inside the parenthesis here)

## Checklist:
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] Some tests are included that test my code

## Screenshots:

image
https://drive.google.com/file/d/1cIyBKvKiUARsX95AARMK-wcsxvj0I8z2/view?usp=sharing

### CODE REVIEW

File: frontend/app/index.txt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { StarRate } from "@/components/StarRate";
+import { CommentRating } from "@/components/CommentRating";
+import { Divider } from "react-native-paper";
+import { useNavigation } from "expo-router";
+import { SongCard } from "@/components/SongBanner";
+import HeaderComponent from "@/components/HeaderComponent";
+import { ReviewButtons } from "@/components/ReviewButtons";
+
+const Index = () => {
+  const navigation = useNavigation(); // Get navigation object
```
Comment (ddusichka) [inline frontend/app/index.txt:20]: do we do anything w this navigation object here?

File: frontend/babel.config.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
-    plugins: ["module:react-native-dotenv"],
+    plugins: ["module:react-native-dotenv", "react-native-paper/babel"],
```
Comment (ddusichka) [inline frontend/babel.config.js:5]: we should remove react-native-dotenv here, it causes the welcome to expo screen of doom

File: frontend/babel.config.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ module.exports = function (api) {
...
+    plugins: ["react-native-paper/babel"],
```
Comment (ddusichka) [inline frontend/babel.config.js:6]: also we can prob delete this second plugins line? seems redundant but idk if I'm missing something

File: frontend/components/AlbumCard.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { View, Text, StyleSheet, Image, TouchableOpacity } from "react-native";
+
+const AlbumCard = () => {
+  const ablumImage = require("@/assets/images/placeholder-image.png");
```
Comment (ddusichka) [inline frontend/components/AlbumCard.tsx:5]: tiny typo
```suggestion
  const albumImage = require("@/assets/images/placeholder-image.png");
```

File: frontend/components/AlbumCard.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    backgroundColor: "#A9A9A9",
+    padding: 10,
+    alignItems: "center",
+  },
+  textContainer: {
+    flex: 1,
+    paddingLeft: 10,
+  },
+  releaseDate: {
+    color: "#434343",
```
Comment (ddusichka) [inline frontend/components/AlbumCard.tsx:48]: just flagging that we should find a way moving forward to standardize these hex codes in one place

File: frontend/components/CommentRating.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { StyleSheet, View, TextInput } from "react-native";
+
+interface StarRateProps {
```
Comment (ddusichka) [inline frontend/components/CommentRating.tsx:4]: nit but can we rename this? it made me think this is the star component but it's just the comment text
```suggestion
interface CommentRatingProps {
```

File: frontend/components/PublishButton.tsx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+interface ReviewCardProps {
+  rating: number;
+  review: string;
+}
+
+const PublishButton: React.FC<ReviewCardProps> = ({ rating, review }) => {
+  const handleClick = () => {
+    console.log("Publishing review");
+    axios
+      .post(`http://10.110.235.22:8080/reviews`, {
```
Comment (ddusichka) [inline frontend/components/PublishButton.tsx:16]: don't forget to swap in the real value here 

`process.env.EXPO_PUBLIC_BASE_URL`

File: frontend/components/StarRate.tsx [hunk 8]
```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { StyleSheet, View, Text, Image } from "react-native";
+import StarRating from "react-native-star-rating-widget";
+
+interface StarRateProps {
+  onRatingChange: (value: number) => void;
+}
+
+const StarRate = ({ onRatingChange }: StarRateProps) => {
```
Comment (ddusichka) [inline frontend/components/StarRate.tsx:9]: this component is super cool!!

File: frontend/components/Visibility.tsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { View, Text, StyleSheet, TouchableOpacity } from "react-native";
+
+const Visibility = () => {
```
Comment (ddusichka) [inline frontend/components/Visibility.tsx:4]: did we decide with the designers if this will still be a feature? i can't remember lol

**Review by ddusichka — COMMENTED** [review]
good stuff!!

File: frontend/components/Visibility.tsx (hunk 9 above)
Comment (gaikwadsid) [reply frontend/components/Visibility.tsx:4]: I think we are continuing with this as a feature.

**Review by ddusichka — DISMISSED** [review]

**Pull request merged**